coffee-module/coffee-module-redisstream/src/main/java/hu/icellmobilsoft/coffee/module/redisstream/annotation/RedisStreamConsumer.java

```diff
@@ -0,0 +1,123 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.coffee.module.redisstream.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.enterprise.util.AnnotationLiteral;
+import javax.enterprise.util.Nonbinding;
+import javax.inject.Qualifier;
+
+import hu.icellmobilsoft.coffee.module.redis.annotation.RedisConnection;
+import hu.icellmobilsoft.coffee.module.redisstream.config.StreamGroupConfig;
+
+/**
+ * Stream consumer pointer and definition. Example:
+ * 
+ * <pre>
```

Comment:
a < és > elemek használata helyett is lehetne használni &# féle kódokat, hogy ne legyen javadoc probléma

Reply:
&#60; -> "<" , &#62; -> ">"

Reply:
Ezzel a formában nincs baj, ez rendes javadoc html szintax. Ott van az ütközés ha `<code>` vagy `<pre>` -ben van használva "<" és ">" karakterek, de ezeket a eclipse szépen replacelné, ahogy látod a belsejében az annotációs @ jelnél